src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -136,6 +136,9 @@ class EmojiPickerMenu extends Component {
                     renderItem={this.renderItem}
                     keyExtractor={item => `emoji_picker_${item.code}`}
                     numColumns={this.numColumns}
+                    removeClippedSubviews
+                    maxToRenderPerBatch={this.numColumns}
```

Comment:
Can you clarify why you're using 3 for windowSize? Not good to have hard coded numbers so good to clarify with a comment.